zipline/src/commonMain/kotlin/app/cash/zipline/Zipline.kt

```diff
@@ -15,10 +15,12 @@
  */
 package app.cash.zipline
 
+import kotlinx.serialization.modules.SerializersModule
+
 expect abstract class Zipline {
   abstract val engineVersion: String
 
-  fun <T : Any> get(name: String, jsAdapter: JsAdapter): T
+  fun <T : Any> get(name: String, serializersModule: SerializersModule): T
```

Comment:
TODO: default this to EmptySerializersModule. Might Just Work with our IR plugin?